examples/monaco-graphql-webpack/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "1.1.1",
   "private": true,
   "license": "MIT",
-  "description": "A simple monaco example with webpack and typescript",
+  "description": "A simple monaco example with webpack and TypeScript",
   "scripts": {
-    "build": "cross-env NODE_ENV=production webpack-cli",
-    "start": "cross-env NODE_ENV=development webpack-cli serve"
+    "bundle": "cross-env NODE_ENV=production webpack-cli",
```

Comment:
renamed to `bundle` because `build` from root fails since I added this example to `workspace` and `postbuild` are executed too late for this example